### ISSUE TITLE
Accessibility confusion regarding message input field

### ISSUE THREAD

**Issue by ndarilek** [vector-im/riot-web#9141]
### Description

I'm a screen reader user confused by the message input field. It seems to be non-keyboard-accessible until I click it, at which point I can enter text. At some point, the ability to enter text goes away until I click the field again.

Is something happening where the field is invisible until some action makes it appear, but that visual appearance isn't being conveyed to my screen reader?

### Version information

- **Platform**: web

For the web app:

- **Browser**: Firefox/Chrome, multiple versions
- **OS**: Linux, Chrome OS
- **URL**: riot.im/app

**Comment by jryans** [comment]
Thanks for the report!

Hmm, I don't see any changes to the DOM when the message input is focused vs. unfocused...

The message input is marked with `role="textbox"`. However, I am not seeing the message input appear in an obvious way the accessibility debugging tools.

We may need to try more debugging here with a screen reader to see what's wrong.

**Comment by lampholder** [comment]
Which screen reader are you using?

**Comment by ndarilek** [comment]
I've had this issue under both Linux/Orca/Firefox, and Chrome OS/Chromevox.

**Comment by lampholder** [comment]
We have an item on our roadmap: ["Simple, stable composer"](https://github.com/vector-im/riot-web/issues/7697) which we might well tackle as part of our upcoming making-messages-editable effort, that will replace the current composer with something as simple as possible to minimise maintenance burden and maximise compatibility.

We should check for screenreader compatibility when we implement that composer.

**Comment by turt2live** [comment]
> Message entry text box contains non-editable text when empty. This makes it impossible to focus this control from screen reader browse mode.

from moz feedback

**Comment by t3chguy** [comment]
This this CIDER or Slate?
If CIDER then https://github.com/vector-im/riot-web/issues/10031 may help

**Comment by turt2live** [comment]
almost certainly Slate, unfortunately (for both the OP and moz). CIDER is a labs option, and the OP is on /app, as is moz. 

iirc we're close to shipping CIDER though, so fixing it for CIDER should be fine imo.

**Comment by t3chguy** [comment]
Closing assuming fixed until told otherwise given we've shipped CIDER
